src/appleseed/renderer/kernel/lighting/sppm/sppmlightingengine.cpp

```diff
@@ -183,6 +183,7 @@ namespace
                 ~0, // max glossy bounces
                 ~0, // max specular bounces
                 ~0, // max volume bounces
+                false,
```

Comment:
Add a comment explaining what this `false` is for (here and in the other places).

Reply:
`// don't clamp roughness` will work? Or `// don't use clamp roughness`?